securedrop/sdconfig.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+MYPY = False
+if MYPY:
+    from typing import List  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        self.ADJECTIVES = _config.ADJECTIVES
+        self.DATABASE_ENGINE = _config.DATABASE_ENGINE
+        self.DEFAULT_LOCALE = _config.DEFAULT_LOCALE
+        self.FlaskConfig = _config.FlaskConfig
+        self.GPG_KEY_DIR = _config.GPG_KEY_DIR
+        self.JOURNALIST_KEY = _config.JOURNALIST_KEY
+        self.JOURNALIST_TEMPLATES_DIR = _config.JOURNALIST_TEMPLATES_DIR
+        self.JournalistInterfaceFlaskConfig = \
```

Comment:
this is surprising, why is it necessary ? and the same question for self.os and all other modules.